sdk/synapse/synapse-artifacts/src/operations/dataFlowDebugSession.ts

```diff
@@ -34,148 +38,24 @@ import type {
   DataFlowDebugSessionExecuteCommandOptionalParams,
   DataFlowDebugSessionExecuteCommandResponse,
   DataFlowDebugSessionQueryDataFlowDebugSessionsByWorkspaceNextResponse,
-  DataFlowDebugCommandResponse,
 } from "../models/index.js";
-import type { RawHttpHeaders } from "@azure/core-rest-pipeline";
```

Comment:
@mpodwysocki I notice that we have a pr to migrate synapse artifact to esm with this big [commit](https://github.com/Azure/autorest.typescript/pull/3103). Could you help me understand if this commit is manually updated or any tool to achieve that?

Service team has the requirement to refresh this SDK but all these changes are reverted which made lint check failed in ci.